cloudant-client/src/test/java/com/cloudant/tests/HttpProxyTest.java

```diff
@@ -217,6 +225,8 @@ public void resetAuthenticator() {
         if (useProxyAuth && useHttpsServer && !okUsable) {
             Authenticator.setDefault(null);
         }
+        // Reset the disabled schemes property
+        System.setProperty("jdk.http.auth.tunneling.disabledSchemes", defaultDisabledList);
```

Comment:
I don't really understand the purpose of this - you don't ever set `jdk.http.auth.tunneling.disabledSchemes` to any different value in this test, so why is the reset needed?